Find Food Trucks SF/Find Food Trucks SF/Main.storyboard

```diff
@@ -48,7 +43,7 @@
                 </tableViewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="U9Q-9i-1u6" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="348" y="1159"/>
+            <point key="canvasLocation" x="348" y="1158.1709145427287"/>
```

Comment:
there was an error with referencing with the old image png, I fixed it and removed the reference then made it refer to new images instead in main.storyboard, Then made it show the nearby icon since it wasn't showing up on tab bar view controller